packages/vue-dot/src/functions/downloadFile/tests/downloadFile.spec.ts

```diff
@@ -0,0 +1,94 @@
+import { downloadFile } from '..';
+
+const file = new File(['hello world'], 'hello_world.txt', { type: 'text/plain' });
+
+// Tests
+describe('downloadFile', () => {
+
+	afterEach(() => {
+		jest.restoreAllMocks();
+	});
+
+	global.URL.createObjectURL = jest.fn();
+	global.URL.revokeObjectURL = jest.fn();
+
+	it('is called correctly', () => {
+		const link = {
+			click: jest.fn(),
+			style: jest.fn(()=>'none')
+		} as any;
+		jest.spyOn(document, 'createElement').mockImplementation(() => link);
+		const appendChilSpy = jest.spyOn(document.body, 'appendChild').mockImplementation();
```

Comment:
```suggestion
		const appendChildSpy = jest.spyOn(document.body, 'appendChild').mockImplementation();
```
Attention aux typos, je te conseille d'installer l'extension "Code Spell Check" pour ça